src/widgetastic/browser.py

```diff
@@ -657,11 +672,11 @@ def move_by_offset(self, x: int, y: int) -> None:
         ActionChains(self.selenium).move_by_offset(x, y).perform()
 
     @retry_stale_element
-    def execute_script(self, script: str, *args, **kwargs) -> Any:
+    def execute_script(self, script: str, *args, silent=False, **kwargs) -> Any:
         """Executes a script."""
         from .widget import Widget
 
-        if not kwargs.pop("silent", False):
```

Comment:
This was also removing `silent` from the `kwargs` which are passed to `selenium.execute_script`.   Doesn't look like this function signature takes kwargs, maybe this isn't covered by unit tests right now?

Reply:
I took another look - `kwargs` is passed to `self.element()`, not execute_script. This isn't an issue.